Gemfile

```diff
@@ -42,6 +42,14 @@ gem 'devise'
 
 gem 'dropzonejs-rails'
 
+gem 'coffee-script-source', '1.8.0'
```

Comment:
There is a trend to use ES6 since 2015 https://thoughtbot.com/blog/replace-coffeescript-with-es6

Reply:
Please move your CoffeeScript to ES6 https://decaffeinate-project.org/repl/